src/Playlist.tsx

```diff
@@ -31,22 +32,38 @@ export const Playlist: React.FC = () => {
 
   if (loading) {
     return (
-      <div>
+      <Box>
         <Spinner />
-      </div>
+      </Box>
     );
   }
 
   if (!data) {
-    return <div>no data in songlist</div>;
+    return <Box>no data in songlist</Box>;
   }
 
   if (data.error) {
-    return <div>404</div>;
+    return <Box>404</Box>;
   }
+  const handleHover = (options: { key: number| null }) => {
+    const { key } = options;
+    if (key || key === 0) setHoverItem(key);
+    if (key === null) setHoverItem(null);
+  };
+
   const listSongs = (list: FetchData) => {
     return list.items.map((song, i) => (
-      <ListItem key={i}>{song.track.name}</ListItem>
+      <ListItem
+        onMouseEnter={() => handleHover({  key: i })}
```

Comment:
handle hover can be done through css - shouldn't be done through javascript. Check out the _hover property like so: 
https://chakra-ui.com/docs/styled-system/features/style-props#pseudo.